app/assets/javascripts/Components/dashboard.jsx

```diff
@@ -3,6 +3,7 @@ import { render } from 'react-dom';
 
 import { Bar } from 'react-chartjs-2';
 
+import {grade_distribution_course_summaries_path} from "../../../javascript/routes";
```

Comment:
You don't need to import this because `Routes` is already set as a global object (in `application.js`). So below, you can just do `Routes.grade...`.